agent/virus_total_agent.py

```diff
@@ -47,6 +49,12 @@ def process(self, message: msg.Message) -> None:
         """
         file_content = file.get_file_content(message)
         if file_content is not None:
+            if (
```

Comment:
is there a reason for not using one **if statement** for all conditions ?

Reply:
They are independent from each other.